CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-<a name="2.0.1"></a>
+<a name="2.0.2"></a>
+# 2.0.2
+
+* Increased the size of nonce and state in OIDCStrategy.
+
```

Comment:
You might want to add the 1.4.7 changes as well so you don't have as hard a time merging when they come together in master.